app/workers/kyc/kycaid/applicant_worker.rb

```diff
@@ -0,0 +1,34 @@
+require 'sidekiq'
+
+module KYC
```

Comment:
For Applicant we simply gather info from user profile and send to KYCAID. At this moment there is no verification from their side, so we just record applicant_id if request was successful and update profile label to verified